cypress/support/config/services.js

```diff
@@ -84,7 +84,9 @@ const genServices = appEnv => ({
     variant: 'default',
     pageTypes: {
       articles: {
-        path: isLive(appEnv) ? undefined : '/amharic/articles/c3rykrrvy19o',
+        path: isLive(appEnv)
+          ? '/amharic/articles/czqverekrldo'
+          : '/amharic/articles/c3rykrrvy19o',
```

Comment:
These new assets are on Test/Local, which is the second line here.  The first line is for live assets, which we don't yet have for these services.

Reply:
Addressed and updated.

Reply:
👍